webrtc.html

```diff
@@ -4106,7 +4106,7 @@
               the candidate that it is derived from. For host candidates, the
               <code>relatedPort</code> is <code>null</code>. This corresponds to
               the <code>rel-port</code> field in <a><code>candidate-attribute</code></a>.</dd>
-              <dt><code><dfn>ufrag</dfn></code> of type <span class=
+              <dt><code><dfn>usernameFragment</dfn></code> of type <span class=
               "idlAttrType"><a>DOMString</a></span>, readonly, nullable</dt>
               <dd>This carries the <code>ufrag</code> as defined in section
```

Comment:
here ufrag is the right thing and doesn't need to be replaced.
But technically https://tools.ietf.org/html/rfc5245#section-15.4 talks about ice-ufrag, not ufrag...